### ISSUE TITLE
2.2.0.0 - Bedrock Dust - Blast Furnace - CTD

### ISSUE THREAD

**Issue by devanchya** [Haggle1996/RevolutionPack#273]
Open up the Blast Furnace, click on the ARROW to open up the various NEI recipes the blast furnace has.
Find the Bedrock Steel option which shows Bedrock Dust.

Click R while hovering over the Dust, and you will get a CTD.

Reproduced twice.
Standard Forge

**Comment by NachtRaben** [comment]
We need logs and any hs-error logs if it was a jvm crash instead of minecraft.

**Comment by Toksyuryel** [comment]
You'll get the same crash anywhere in NEI. It's the same Grinder recipe issue that we're waiting on v11 for.

**Comment by Haggle1996** [comment]
Possible duplicate of #205, but need crash log.

**Comment by devanchya** [comment]
duplicate, closing.
